x/bank/keeper/keeper.go

```diff
@@ -412,6 +417,58 @@ func (k BaseKeeper) BurnCoins(ctx context.Context, moduleName string, amounts sd
 	return nil
 }
 
+// SubtractCoins subtracts coins from the balance of an account
+// NOTE(Heimdall-v2): required in heimdall for fee withdrawal in topup module
+func (k BaseKeeper) SubtractCoins(ctx context.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
```

Comment:
Why not added to the interface?

Reply:
There's no need for this function, there are functions already for sending tokens to and from a module...in this case the TopUp module would need to be removed from the `BlockedAddresses` list in the app and have its permissions updated so it can send/receive tokens

Reply:
Done [1cbb41b](https://github.com/0xPolygon/cosmos-sdk/pull/5/commits/1cbb41b4cb73390c58ced14d1e5f76288a1802a3)